src/arch/timer.h

```diff
@@ -43,20 +43,20 @@ extern timer_uint timer_perf_freq;
 
 inline timer_uint timer_new(void)
 {
-	LARGE_INTEGER __start_time;
-	QueryPerformanceCounter(&__start_time);
-	return (timer_uint)__start_time.QuadPart;
+	LARGE_INTEGER start_time;
```

Comment:
It' ok, but we can be safe with `__` anyhow.

Reply:
To be on the safe side, I think we should avoid identifiers starting with an underscore (https://stackoverflow.com/a/25090719). If you agree, my policy is to change such identifiers gradually over time.

Reply:
Yes, it's fine for me to drop the starting `__`, provided that in our compilation flow we are 100% sure that all symbols in the libraries are hidden from the users of the library (except for the API)!
I have lost control on this particular aspect, so if we set the default visibility to hidden, you can proceed safely!

Reply:
The variables I renamed are declared in their respective block scopes, therefore, if I am not mistaken, they shouldn't create conflicts with outside symbols (including the ones declared by library users). Currently, the symbols visibility problem hasn't been addressed, I'll open an issue about it!

Reply:
Yup! I was talking in general, not referred to these scope variables, which is perfectly fine.
Thanks for the issue, that should be easy to fix (and I'll comment there later).